samples/graph-change-notification-team-channel/csharp/ChangeNotification/Model/Configuration/BotConfiguration.cs

```diff
@@ -0,0 +1,43 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+namespace ChangeNotification.Model.Configuration
+{
+    public class BotConfiguration
```

Comment:
We don't have bot for this application, so we can rename this class as Configuration Or ApplicationConfiguration.